docs/ArchLinux.md

```diff
@@ -0,0 +1,96 @@
+# Pam Duress on Arch
+
+This is a guide on how to use pam duress on Arch linux in support of
+[Issue 29](https://github.com/nuvious/pam-duress/issues/29) on GitHub
+submitted by [Dusan Lesan](https://github.com/DusanLesan).
+
+## Installation 
+
+### Dependencies
+
+For the dependencies arch packages the development libraries with opensll
+so your dependencies look more like the below:
+
+```bash
+sudo pacman -u
```

Comment:
This option does not exist. I guess you wanted `pacman -Su` or  `pacman -Syu`

Reply:
Good catch. Should be resolved in the most recent push.